ground-server/Dockerfile

```diff
@@ -14,16 +15,22 @@ RUN npm install --frozen-lockfile
 RUN npm run build
 
 # Stage 2: Serve the app using Nginx
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +1 to +17]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->We no longer need a build step. We can just run npm install in the main Node environment, since we need node to serve the Next.js application<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/37?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->